outputs.tf

```diff
@@ -1,20 +1,15 @@
 locals {
   grafana_endpoint = join("", module.grafana.*.grafana_endpoint)
 }
+
 output "cluster_id" {
   description = "Cluster ID of Thanos deployment"
   value       = local.postfix_name
 }
 
 output "thanos_space_id" {
```

Comment:
Does this output become redundant now?

Reply:
We still need an internal identifier if the prefix is not provided as input, otherwise we will get regional conflicts between customers. It might not make sense to export the "naked" value if we are exporting all relevant endpoints. I expect some points releases over the next few days to address this.

I'm also creating a thanos-on-hsdp example project which will consume this new version. This should make the initial conversation with customers who are interested in consuming this much easier.

Reply:
You could use the module I created in innersource as a blueprint for this. It has a lot of the _other_ stuff in there already.

Such as adding routes and configuring a security layer with IAM.